client/web/src/site-admin/SiteConfigurationChangeList.tsx

```diff
@@ -95,6 +95,14 @@ const SiteConfigurationHistoryItem: FC<SiteConfigurationHistoryItemProps> = ({ n
     const icon = open ? mdiChevronUp : mdiChevronDown
     const [removedLines, addedLines] = linesChanged(node.diff)
 
+    const editedBy = node.author ? (
+        <Link to={`/users/${node.author.username}`} className="text-truncate">
+            {node.author.displayName}
+        </Link>
+    ) : (
+        'SITE_CONFIG_FILE edited by a site admin'
```

Comment:
I have to admit that I misread this line 5 times, because I interpreted it as "site config edited by site-admin" and then went "well, tell me which site admin then?".

Maybe something like this?

```suggestion
        'Site configuration updated through SITE_CONFIG_FILE environment variable'
```

Reply:
Hmm, I agree. How about:

`Site configuration file as set via SITE_CONFIG_FILE environment variable updated by a site admin`

Reply:
Let's make it shorter: `Site configuration file set in SITE_CONFIG_FILE environment variable updated`

I think that gets the point across

Reply:
Fixed. 
<img width="1186" alt="image" src="https://user-images.githubusercontent.com/2682729/228141183-5b6d1c52-296d-4445-a7d9-49934c97f396.png">